frontend/src/components/Landing/signin-button.test.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Test file for the SignInButton component.
+ */
+
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+
+import SignInButton from './signin-button';
+import { SIGN_IN } from '../../constants/routes';
+
+// Mock the push function from react-router-dom's useHistory.
+const mockHistoryPush = jest.fn();
+jest.mock('react-router-dom', () => ({
+  useHistory: () => ({
+    push: mockHistoryPush
+  })
+}));
+
+// SignInButton tests.
```

Comment:
this line shouldn't be necessary, description right below says the same thing.
```suggestion
```

Reply:
Good point, it was removed. 👍